src/extension/SettingsPanel.ts

```diff
@@ -75,6 +75,23 @@ export class SettingsPanel {
     this.dispose()
   }
 
+  /**
+   * returns the name of the conf file to edit, if such file exists and has a name
+   * empty string otherwise
+   * @param editConfFile conf file to edit
+   * @returns string name
+   */
+  private static _getConfFileName(
```

Comment:
You did well, putting it in a separate function.
But, please stick to its documentation: this function returns the filename.
Please don't add the `: ` as a prefix (You can handle it in the `_openNewPanel` function by checking the returned value).

Reply:
`if (confFileName) {
      confFileName = ': ' + confFileName
    }`
?

Reply:
Yes :+1: